### ISSUE TITLE
P0f will not start

### ISSUE THREAD

**Issue by louismunro** [inverse-inc/fingerbank#31]
When trying to start PF, p0f will not start with the following error:

```
[-] SYSTEM ERROR : Cannot open '/usr/local/fingerbank/conf/fingerbank-p0f.fp' for reading.
        Location : read_config(), readfp.c:392
      OS message : No such file or directory

p0f|not started
```

This is with fingerbank-2.0.0-3.1.noarch

**Comment by julsemaan** [comment]
Because you don't have fingerbank 2.0

**Comment by louismunro** [comment]
I do.
As stated I have fingerbank-2.0.0-3.1.noarch.

**Comment by dwlfrth** [comment]
Should most probably be fixed with latest package (2.0.0-12)
Was missing the fingerbank-p0f.fp file
